### ISSUE TITLE
Live demo of new route fixes

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-hashed-urls#11]
Refs:
- #8
- #9
- #10

**Comment by simonw** [comment]
`datasette-publish-vercel` doesn't have a mechanism for enabling `crossdb=True`.

**Comment by simonw** [comment]
Publishing that demo using:
```
datasette publish vercel fixtures.db fixtures2.db \
  --crossdb \
  --branch 0.61a0 \
  --project datasette-hashed-urls-preview \
  --install https://github.com/simonw/datasette-hashed-urls/archive/be554804c0a68e21c3cd52e9aebb287d10d2179e.zip \
  --about 'datasette-hashed-urls demo' \
  --about_url 'https://github.com/simonw/datasette-hashed-urls/issues/11'
```

**Comment by simonw** [comment]
https://datasette-hashed-urls-preview.vercel.app/_memory?sql=select+*+from+fixtures.facetable+union+all+select+*+from+fixtures2.facetable

**Comment by simonw** [comment]
Leaving this issue open for the moment - I'll close it when I ship this as the default demo at https://datasette-hashed-urls.vercel.app/ - which I'll do after I ship a non-alpha release of the plugin with this fix, which I'll do after I ship Datasette 0.62 non-alpha.

**Comment by simonw** [comment]
Shipping a fresh demo:
```
datasette publish vercel fixtures.db fixtures2.db \
  --crossdb \
  --project datasette-hashed-urls-preview \
  --install datasette-hashed-urls \
  --about 'datasette-hashed-urls demo' \
  --about_url 'https://datasette.io/plugins/datasette-hashed-urls'
```
